cocos/core/pipeline/render-additive-light-queue.ts

```diff
@@ -54,6 +54,13 @@ interface IAdditiveLightPass {
 
 const _lightPassPool = new Pool<IAdditiveLightPass>(() => ({ subModel: null!, passIdx: -1, dynamicOffsets: [], lights: [] }), 16);
 
+interface InstancedOrBatchedLightPass {
```

Comment:
Just reuse `IAdditiveLightPass` and `_lightPassPool`, only use `dynamicOffsets` and `lights` for instanced or batched queue

Reply:
sure